scripts/sdk/base-manifest.json

```diff
@@ -2,6 +2,17 @@
     "name": "hydrogen-view-sdk",
     "description": "Embeddable matrix client library, including view components",
     "version": "0.0.5",
-    "main": "./hydrogen.es.js",
+    "main": "./hydrogen.cjs.js",
+    "exports": {
+        ".": {
+            "import": "./lib-build/hydrogen.es.js",
+            "require": "./lib-build/hydrogen.cjs.js"
+        },
+        "./paths/vite": "./paths/vite.js",
```

Comment:
This export isn't used but it's part of the `./scripts/sdk/transform-paths.js ./src/platform/web/sdk/paths/vite.js ./target/paths/vite.js` stuff in `build.sh` that's currently commented out.